### PR TITLE
Contrib/CDS Cofense Vision 100

### PR DESCRIPTION
**Pull request by crestdatasystems**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Same PR as #21551 
- Search for suspicious and malicious emails.
- Create and approve jobs against emails to quarantine threats in mailboxes.
- Create quarantine jobs to remove suspicious emails from mailboxes using automation.
- Download original emails and get suspicious attachments for additional analysis.
- Add IOCs to be used in searching and quarantine job operations. 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CofenseVision/Integrations/CofenseVision/CofenseVision.yml [hunk 1]
```diff
@@ -0,0 +1,2770 @@
...
+        description: A description of the malicious email.
+        type: String
+      - contextPath: Email.Malicious.Vendor
+        description: The vendor who reported the email as malicious.
+        type: String
+      - contextPath: File.SHA256
+        description: The SHA256 hash of the file.
+        type: String
+      - contextPath: File.MD5
+        description: The MD5 hash of the file.
```
Comment (dorschw) [inline Packs/CofenseVision/Integrations/CofenseVision/CofenseVision.yml:2737]: regarding these fields, are they available on Cofense, and can be mapped in the IP indicator? Same for the rest of the standard indicator objects.

Comment (crestdatasystems) [reply Packs/CofenseVision/Integrations/CofenseVision/CofenseVision.yml:2737]: Yes, we've mapped some of the standard fields to common indicators fields.

File: Packs/CofenseVision/Integrations/CofenseVision/CofenseVision.py [hunk 2]
```diff
@@ -0,0 +1,2558 @@
...
+    """
+    search_id = args.get('id', '')
+    validate_search_id(search_id)
+
+    page = arg_to_number(args.get('page', 0), arg_name='page')
+    size = arg_to_number(args.get('size', 50), arg_name='size')
+
+    if int(page) < 0:  # type: ignore
+        raise ValueError(ERROR_MESSAGE['INVALID_PAGE_VALUE'])
+    validate_page_size(size)
```
[the remark this answers is not in the document]
Comment (crestdatasystems) [reply Packs/CofenseVision/Integrations/CofenseVision/CofenseVision.py:1972]: Yes, we do have validation: https://github.com/demisto/content/pull/22018/files/a9d32aa1b1895f130c29d01e228690d17174aac5#diff-4662391038595b7b79689bdc4d2af75f9481c9bb8297b3d121d31f7acbba8a07R771

File: Packs/CofenseVision/Integrations/CofenseVision/CofenseVision.py [hunk 3]
```diff
@@ -0,0 +1,2558 @@
...
+            dbot_score=dbot_score
+        )
+    elif threat_type == "sha256":
+        standard_context = Common.File(  # type: ignore
+            sha256=threat_value,
+            dbot_score=dbot_score
+        )
+    else:
+        standard_context = dbot_score  # type: ignore
+    return standard_context
```
[the remark this answers is not in the document]
Comment (crestdatasystems) [reply Packs/CofenseVision/Integrations/CofenseVision/CofenseVision.py]: Updated

**Pull request merged**